app/assets/stylesheets/components/_navbar.scss

```diff
@@ -1,69 +1,93 @@
 /* Main navbar */
-.navbar-wagon {
+.navbar-waterfall {
   box-shadow: 0 1px 5px 0 rgba(0,0,0,0.07),0 1px 0 0 rgba(0,0,0,0.03);
-  background: white;
+  background: $dark-gray;
   transition: all 0.3s ease;
-  height: 70px;
+  height: 90px;
   padding: 0px 30px;
   display: flex;
   align-items: center;
   justify-content: space-between;
+  border-top: 20px solid black;
 }
 
 /* Logo */
-.navbar-wagon-brand img {
-  height: 50px;
+.navbar-waterfall-brand img {
+  height: 40px;
+  padding: 2px;
 }
 
 /* User Avatar */
-.navbar-wagon .avatar {
+.navbar-waterfall .avatar {
   height: 35px;
   width: 35px;
   border-radius: 50%;
+  border: 1px solid $green;
 }
 
 /* Right section */
-.navbar-wagon-right {
+.navbar-waterfall-right {
   display: flex;
   align-items: center;
   justify-content: space-between;
 }
 
 /* Basic navbar item */
-.navbar-wagon-item {
+.navbar-waterfall-item {
   cursor: pointer;
   padding: 0 20px;
 }
 
+/* Navbar button */
+.navbar-button-green {
```

Comment:
Could we discuss tomorrow if we should rename this navbar-button-green? To button-green or button and just make a general component out of it? So we can use it later, adapted in different parts of our site. I'll merge for now though.